QEfficient/utils/generate_inputs.py

```diff
@@ -15,10 +15,12 @@ class InputHandler:
     def __init__(self, tokenizer, input_str, prompt_len, ctx_len):
         """
         Initialization
-        :param model_name: str
-        :param input_str: List[str]
-        :param prompt_len: int
-        :param ctx_len: int
+        ---------
+        
+        :model_name: str. Hugging Face Model Card name, Example: [gpt2].

```

Comment:
model_name is not an input here, correct it for tokenizer.
:param tokenizer: Union[PreTrainedTokenizer, PreTrainedTokenizerFast].  model tokenizer.